src/Hng.Application/Services/ProductService.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Hng.Application.Interfaces;
+using Hng.Application.Models;
+using Hng.Domain.Entities;
+using Hng.Infrastructure.Context;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hng.Application.Services
+{
+    public class ProductService : IProductService
```

Comment:
All services should be in `Hng.Infrastructure` project. I'll stop reviewing until you have fixed this

Reply:
Done